### PR TITLE
feat!: increase line height and font sizes in font scale

### PR DESCRIPTION
**Pull request by frshwtr**
## Describe your changes

The scale has been offset by -1, resulting in font sizes being increased on the current scale.

Another PR will follow to update usage in o3-foundation and usage in other libraries.

## Checklist before requesting a review

- [ ] I have applied `percy` label for o-[COMPONENT] or `chromatic` label for o3-[COMPONENT] on my PR before merging and after review. Find more details in [CONTRIBUTING.md](https://github.com/Financial-Times/origami/blob/main/CONTRIBUTING.md#pull-requests-and-visual-regression-tests)
- [ ] If it is a new feature, I have added thorough tests.
- [ ] I have updated relevant docs.
- [ ] I have updated relevant env variables in Doppler.

### CODE REVIEW

**Comment by frshwtr** [comment]
I would like to make [dde3edb](https://github.com/Financial-Times/origami/pull/1805/commits/dde3edb1d1d94aaaed330e1d74c4f0c61110adad) a breaking commit. I thought initially this wouldn't be necessary, but as this results in the deletion of scale `10`, it may be necessary.

**Comment by frshwtr** [comment]
The for-everyone-website documentation for scales will need updating.

File: apps/dictionary/tokens/core/base/typography.json [hunk 1]
```diff
@@ -151,105 +151,105 @@
...
+          "value": "28",
           "type": "lineHeights"
         },
         "3": {
-          "value": "28",
+          "value": "32",
           "type": "lineHeights"
         },
         "4": {
           "value": "32",
```
Comment (akomiqaia) [inline apps/dictionary/tokens/core/base/typography.json:166]: why do we have two 32s? does this need to be 36?

File: apps/dictionary/tokens/internal/base/typography.json [hunk 2]
```diff
@@ -151,105 +151,105 @@
...
+          "value": "28",
           "type": "lineHeights"
         },
         "3": {
-          "value": "28",
+          "value": "32",
           "type": "lineHeights"
         },
         "4": {
           "value": "32",
```
Comment (akomiqaia) [inline apps/dictionary/tokens/internal/base/typography.json:166]: same here: we 32 twice

File: apps/dictionary/tokens/core/base/typography.json (hunk 1 above)
Comment (frshwtr) [reply apps/dictionary/tokens/core/base/typography.json:166]: In the old scale, values `4` and `5` are both `32`. There are a few repeats like this for line height. The values on [this ticket](https://financialtimes.atlassian.net/jira/software/c/projects/OR/boards/1859?selectedIssue=OR-830) show this should be the case.

**Review by akomiqaia — APPROVED** [review]

File: components/o3-foundation/MIGRATION.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+## Migrating from v1 to v2
+
+v2 introduces changes to the scale for Metric2:
+
+* `--o3-font-lineheight-metric2-10` and `--o3-font-size-metric2-10` has been removed.
+* Scale for font size and line height has been offset by `-1`, meaning the value of scale `--o3-font-size-metric2-1` is
+  now scale `--o3-font-size-metric2-0` and so on.
+* Introduced `--o3-font-size-metric2-negative-3` scale.
+
+This version will not break unless you have used scale `--o3-font-lineheight-metric2-10`
```
Comment (notlee) [inline components/o3-foundation/MIGRATION.md:12]: This is a visually breaking change though, as we don't want to blanket increase type sizes across the board. Users should switch to the lower size at this point.

**Pull request merged**